src/coresyftools/coresyftools/tests/test_tool.py

```diff
@@ -297,3 +298,16 @@ def test_can_handle_missing_command_placeholders(self):
         })
         with self.assertRaises(MissingCommandPlaceholderForOption):
             CoReSyFTool(self.runfile)
+        
+    def test_can_use_custom_manifest_name(self):
+        manifest = self.manifest.copy()
```

Comment:
Perhaps you should add here a docstring.

Reply:
Fixed.